helpers/product/product.go

```diff
@@ -287,3 +287,33 @@ func GetPackageByKeyName(
 
 	return packages[0], nil
 }
+
+// GetPriceIDByPackageIdandLocationGroup return the priceids of the items according to the location group ids of the datacenter u specify
+func GetPriceIDByPackageIdandLocationGroup(addon string, listofpriceids []int, sess *session.Session, packageid int) (int, error) {
```

Comment:
I don't see where the location group is being specified

Also, the session should be the first arg by convention.